lib/controllers/index.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (crowi, app) => ({
+  Admin: require('./admin')(crowi, app),
+  Attachment: require('./attachment')(crowi, app),
+  Backlink: require('./backlink')(crowi, app),
+  Bookmark: require('./bookmark')(crowi, app),
+  Comment: require('./comment')(crowi, app),
+  Installer: require('./installer')(crowi, app),
+  Login: require('./login')(crowi, app),
+  Logout: require('./logout')(crowi, app),
+  Me: require('./me')(crowi, app),
+  Notification: require('./notification')(crowi, app),
+  Page: require('./page')(crowi, app),
+  Revision: require('./revision')(crowi, app),
+  Search: require('./search')(crowi, app),
+  Share: require('./share')(crowi, app),
+  ShareAccess: require('./shareAccess')(crowi, app),
+  Slack: require('./slack')(crowi, app),
+  User: require('./user')(crowi, app),
```

Comment:
Can you let me know the reason why do you use `UpperCamel` here for Objects?
It is not clearly specified but we conventionally use `lowerCamel` for normal Objects right?

Reply:
We are using `UpperCamel` case to name application-level objects such as models and events. I followed them.